### DISCUSSION TITLE
How to run OSP Suite ?

### DISCUSSION THREAD

**Discussion by janakrw** [Open-Systems-Pharmacology/Forum#1687] [Questions & Problems]
### Discussed in https://github.com/Open-Systems-Pharmacology/Forum/discussions/1420

<div type='discussions-op-text'>

<sup>Originally posted by **egmnb** December 27, 2022</sup>
I am not only new for OSP but also Github. I have checked most of the things but could not find proper answer that would work for me.

Is there any detail documentation to how to run OSP ?

Accept my Apologise in advance.</div>

**Comment by Yuri05** [comment]
Main documentation: https://docs.open-systems-pharmacology.org/
Tutorials: http://tutorials.open-systems-pharmacology.org/

**Reply by OneWay289** [reply]
It appears to me that the documentation is somewhat out of date.  For example, I viewed "Video 8.  How to set up a simulation?" and it says it is for PK-Sim Version 7.3.0. (https://www.youtube.com/watch?v=JndcMlv47sY) Everything makes sense and seems to match version 11.3; however, there is no "Observers" category. I am lost as to what an Observer is and how I enter one.  The documentation at this address also doesn't list Observers: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations

I've tried several things but nothing works.  I am working with a simple molecule administered orally.  Any help you can give me would be greatly appreciated!

Thanks.

**Comment by OneWay289** [comment]
I figured out my problem. I had the screen setting on my computer above the recommended 125% as my eyesight is not good.  This took the "OK' button off the screen so I couldn't see it. The old video actually helped me as I found that there was an "OK" that I couldn't see.

A simple problem that I should have thought of earlier, but hopefully my blunder will help someone else in the future.

**Reply by Yuri05** [reply]
This should not happen anyway... what was your font size (150% ? more?) and in which program window you could not see the OK button? Also which version of PK-Sim are you using?

**Reply by OneWay289** [reply]
Thanks for the reply, I just noticed it.  I'm using PK-Sim 11.3 Build 208.  Yes, I set my computer to 150% scale in settings/system/display because I have difficulty reading it at lower scales (e.g., 125%).  I use this for all programs except for PK-Sim.  

I am not having problems with the Observer when I use 125% - I can see the "OK" fine and just click on that because I don't have an Observer.  I am working my way through the Esq training which is very helpful.

Thanks again for your reply.
